e2e/utils/command-utils.ts

```diff
@@ -70,26 +71,33 @@ export function runCommand(
 ): string {
   const { failOnError, ...childProcessOptions } = options ?? {};
   try {
-    const r = execSync(command, {
+    const r = execSync(`${command}${isVerbose() ? ' --verbose' : ''}`, {
       cwd: tmpProjPath(),
-      stdio: ['pipe', 'pipe', 'pipe'],
+      stdio: 'pipe',
       env: {
         ...getStrippedEnvironmentVariables(),
         ...childProcessOptions?.env,
         FORCE_COLOR: 'false',
-      },
+      } as any,
```

Comment:
Do we need type here? It should be automatically inferred based on the `encoding`.

If yes, please use `NodeJS.ProcessEnv` instead `any

Reply:
It was throwing an error, which is not any more 🤔